upup/pkg/fi/cloudup/awstasks/launchconfiguration.go

```diff
@@ -36,12 +36,13 @@ type LaunchConfiguration struct {
 
 	UserData *fi.ResourceHolder
 
-	ImageID            *string
-	InstanceType       *string
-	SSHKey             *SSHKey
-	SecurityGroups     []*SecurityGroup
-	AssociatePublicIP  *bool
-	IAMInstanceProfile *IAMInstanceProfile
+	ImageID                    *string
+	InstanceType               *string
+	SSHKey                     *SSHKey
+	SecurityGroups             []*SecurityGroup
+	AdditionalSecurityGroupIDs []string
```

Comment:
This one it is fine to call `AdditionalSecurityGroupIDs`, because (1) this code is AWS only and (2) this field isn't exposed to end users, so it doesn't really much matter what we call it :-)